docs/source/queries/timeouts.md

```diff
@@ -0,0 +1,42 @@
+# Query timeouts
+
+Query execution time can be limited by setting a request timeout. If a query does not complete
+in the given time, it is interrupted and `QueryError::RequestTimeout` is returned. Timeout
```

Comment:
I think that "it is interrupted" may be misleading - the driver simply stops actively waiting for its response. "Interrupted" may suggest that the request will also stop being processed by the database server, and that's not the case. Let's rephrase to emphasize that the error is returned by the driver immediately after the timeout passes, so that application logic can continue operating, but the query may still be in progress on the server.

Reply:
Perhaps it's also worth mentioning, as a side note, that for custom server-side timeouts, one can use a `USING TIMEOUT <duration>` directive supported in ScyllaDB (but not in Cassandra). Ref: https://github.com/scylladb/scylladb/blob/master/docs/cql/cql-extensions.md#using-timeout